tests/plugins/divarea/manual/detached.md

```diff
@@ -0,0 +1,37 @@
+@bender-tags: 4.17.0, bug, 4462, divarea
+@bender-ui: collapsed
+@bender-ckeditor-plugins: divarea,toolbar,basicstyles,link,format,sourcearea,elementspath,undo
+
+
+1. Open dev console
+
+2. Click "Toggle" button twice. To hide and show editor.
+
+	**Expected:** Editor shows with same data. Content is editable. Elementspath (at the bottom is updated).
```

Comment:
When we have 3 or more things to check it's more readable in form of bullet points:
```suggestion
  **Expected:**

  * Editor shows with same data.
  * Content is editable.
  * Elementspath (at the bottom is updated).
```

Also I recommend using double spaces (`  `) instead of tabs as they allow for better control over the indentation afterwards.

Same for all other manual tests.

Reply:
I split it into three separate steps.

I've checked at the beginning other manual tests and they also use tabs. Also, find in our code style guide (but in notion)  
https://www.notion.so/Code-Style-cb28fa1931bf48b6988ea3bb3990f566#393baa32179d49ef9ad4f3be923afe85

Reply:
So we decide (@Dumluregn & @f1ames) to allow spaces in bender tests. It's the only way to properly format test descriptions in bender.